src/app/parcels/add/[id]/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { Metadata } from "next";
+import React from "react";
+import AddParcelForm from "@/app/parcels/add/AddParcelForm";
+
+type AddParcelParameters = {
+    params: { id: string };
+};
+
+const AddParcels: ({ params }: AddParcelParameters) => React.ReactElement = ({ params }) => {
+    return (
+        <main>
+            <AddParcelForm id={params.id} />
+        </main>
+    );
+};
+
+export const metadata: Metadata = {
+    title: "Add Parcels",
+};
+
+export default AddParcels;
```

Comment:
Will it be a problem that the exported component in both `[id]\page` and `add\page` are called `AddParcels`?

Reply:
Don't think so? Will change anyway though in case but I think the way the next navigation works with pages means it would be ok.